app/components/meeting-form.js

```diff
@@ -39,6 +41,10 @@ export default class MeetingForm extends Component {
   *loadData() {
     if (this.zitting.get("id")) {
       this.bestuursorgaan = yield this.zitting.get("bestuursorgaan");
+      const specialisedBestuursorgaa = yield this.bestuursorgaan.get("isTijdsspecialisatieVan");
```

Comment:
there's a typo here `specialisedBestuursorgaa`
```suggestion
      const specialisedBestuursorgaan = yield this.bestuursorgaan.get("isTijdsspecialisatieVan");
```